source/retryable-reads/tests/unified/ExceededTimeLimit.yml

```
@@ -0,0 +1,76 @@
+description: "ExceededTimeLimit is a retryable read"
+
+schemaVersion: "1.3"
+
+runOnRequirements:
+  - minServerVersion: "4.2"
+    topologies: [replicaset, sharded, load-balanced]
+    auth: true
```

```suggestion
  - minServerVersion: "4.0" # Require server 4.0 for `failCommand`.
```

I expect all topologies support retryable reads.

The [retryable reads test README](https://github.com/mongodb/specifications/blob/master/source/retryable-reads/tests/README.rst#L24-L27) suggests 4.0 is needed for `failCommand`:

> The spec specifies 3.6 as the minimum server version: however, failCommand is not supported on 3.6, so for now, testing requires MongoDB 4.0

---

I believe >=4.1.7 is required for `failCommand` on sharded topologies. Most of the legacy retryable reads tests condition the server version required based on topology ([example](https://github.com/mongodb/specifications/blob/82da02ba96d5d3ec4f48f6d2d4f2d0ebb1dce241/source/retryable-reads/tests/legacy/aggregate.json#L2)). I suggest we use the same requirements here.

---

Thanks Isabel, I implemented your suggestion in this commit: https://github.com/mongodb/specifications/pull/1478/commits/61faa8f091accca216f95f5b935518a9685a8c71